app/src/main/res/layout/home_change_address.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:id="@+id/change_address_layout"
+        android:layout_height="wrap_content"
+        android:layout_width="match_parent"
+        android:layout_marginTop="@dimen/base_margin_sextuple"
+        android:layout_marginBottom="@dimen/base_margin_triple"
+        android:orientation="vertical">
+
+    <TextView
```

Comment:
I've been thinking about making one generic H5-item layout to represent these 'Title 2'-items from the designs, and reusing that. Under that scheme, we could return one 'Title 2'-item with the appropriate text, and have a separate item for the 'Change your address'-thing. This would also remove the wrapping `LinearLayout`. What do you think about that? 🙂

Reply:
Sounds good!